webapp/locales/en.json

```diff
@@ -269,11 +269,9 @@
       "name": "Download Data"
     },
     "deleteUserAccount": {
-      "name": "Delete data",
-      "contributionsCount": "Delete my {count} posts",
-      "commentedCount": "Delete my {count} comments",
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-12-07T16:07:17Z" title="Saturday, December 7th 2019, 5:07:17 pm +01:00">Dec 7, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -269,11 +269,9 @@
       "name": "Download Data"
     },
     "deleteUserAccount": {
-      "name": "Delete data",
-      "contributionsCount": "Delete my {count} posts",
-      "commentedCount": "Delete my {count} comments",
```


Bist du dir sicher, dass die Übersetzungen `contributionsCount` und `commentedCount` nicht mehr benötigt werden? Wenn ich den Befehl (s.u..) eingebe, dann erhalte ich einen Treffer im Code.

-----


Are you sure that `contributionsCount` and `commentedCount` are not needed anymore?

E.g. when I run:
```sh
git grep 'deleteUserAccount.commentedCount'
```

I see one search result in `webapp/components/DeleteData/DeleteData.vue`

Reply:
> [<img alt="erhardtstiefel" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/erhardtstiefel) **Authored by [erhardtstiefel](https://github.com/erhardtstiefel)**
_<time datetime="2019-12-07T16:24:40Z" title="Saturday, December 7th 2019, 5:24:40 pm +01:00">Dec 7, 2019</time>_

---


Yes, I wrote another Pull Request 
Update DeleteData.vue #2447